api/CcsSso.Core.DbMigrations/Scripts/DataScripts/20220119193322_InsertCountryDetailsData.sql

```diff
@@ -211,3 +211,12 @@ INSERT INTO public."CountryDetails"("Code", "Name", "CreatedUserId", "LastUpdate
 INSERT INTO public."CountryDetails"("Code", "Name", "CreatedUserId", "LastUpdatedUserId", "CreatedOnUtc", "LastUpdatedOnUtc", "IsDeleted") VALUES ('ZM','Zambia',0, 0, now(), now(), false);
 INSERT INTO public."CountryDetails"("Code", "Name", "CreatedUserId", "LastUpdatedUserId", "CreatedOnUtc", "LastUpdatedOnUtc", "IsDeleted") VALUES ('ZW','Zimbabwe',0, 0, now(), now(), false);
 
+UPDATE public."CountryDetails" 
```

Comment:
Dont add update script to initial data script file. Just correct the file by removing unwanted countries and adding the new countries.